src/styles/blocks/card.scss

```
@@ -0,0 +1,92 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: 300ms;
+  padding: 0 16px;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 32px 3px 40px;
+  }
+
+  &__device {
+    margin: 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__productCode {
+    margin: 0;
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__amount {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+    margin: 0;
+  }
+
+  &__sum {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    color: #fff;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    background: $main-blue-color;
+    border-radius: 5px;
+    width: 166px;
+    height: 40px;
+    text-decoration: none;
+    transition-duration: 300ms;
+
+    &:hover {
+      border-radius: 5px;
+      border: 1px solid $main-blue-color;
+      background: #fff;
+      color: $main-blue-color;
+      width: 166px;
+      height: 40px;
```

Do not repeat your self
```suggestion
      border: 1px solid $main-blue-color;
      background: #fff;
      color: $main-blue-color;
```